paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkCatalog.java

```diff
@@ -357,7 +374,9 @@ public void alterTable(
         List<SchemaChange> changes = new ArrayList<>();
         if (null != tableChanges) {
             List<SchemaChange> schemaChanges =
-                    tableChanges.stream().map(this::toSchemaChange).collect(Collectors.toList());
+                    tableChanges.stream()
+                            .flatMap(tableChange -> (toSchemaChange(tableChange).stream()))
```

Comment:
`(toSchemaChange(tableChange).stream())`
The `()` is redundant.

Reply:
Thanks a lot, updated.